packages/components/src/styles/Button.css.ts

```diff
@@ -0,0 +1,195 @@
+import type { RecipeVariants } from '@vanilla-extract/recipes';
+
+import { vars } from '@launchpad-ui/vars';
+import { recipe } from '@vanilla-extract/recipes';
+
+import { variants as buttonGroup } from './ButtonGroup.css';
+
+const variants = recipe({
+  base: {
```

Comment:
Just a personal thing, nothing to do: I do love my CSS, especially after seeing this. 😅

Reply:
Is it at least easier to parse than the current button styles? This component probably has the most variants but I want to make sure VE doesn't intimidate folks too much.